### PR TITLE
Does not block when consuming has an error now

### PR DESCRIPTION
**Pull request by kngu9**

### CODE REVIEW

File: exporter/consumer.go [hunk 1]
```diff
@@ -186,20 +186,6 @@ func NewConsumer(config ConsumerConfig) (*Consumer, error) {
 	if err != nil {
 		return nil, errors.Annotate(err, "unable to create new kafka client")
 	}
-	topics, err := client.Topics()
```
Comment (cmars) [inline exporter/consumer.go:189]: No longer necessary? What happens if the topic doesn't exist now? Does that come through c.group.Errors() (and have you confirmed this)?

File: exporter/exporter.go [hunk 2]
```diff
@@ -145,49 +144,28 @@ func main() {
...
-			nextTime = (time.Duration(math.Exp2(float64(tries))) * time.Millisecond) + time.Duration(rand.Intn(100))
-			if nextTime > maxBackoff {
-				log.Printf("next timer %+v surpasses the max backoff time of %+v, setting to max backoff time\n", nextTime.String(), maxBackoff.String())
-				nextTime = maxBackoff
-			}
+	consumers := make([]*Consumer, 0)
+	for _, topicConfig := range config.Topics {
+		consumer, err := startConsumer(ctx, config.kafkaBrokers(), tlsConfig, httpClient, topicConfig)
+		if err != nil {
+			exitTime := (5 * time.Second) + time.Duration(rand.Intn(100))
```
Comment (cmars) [inline exporter/exporter.go]: I don't see much value in jittering the exit time by 100ns.

**Review by cmars — COMMENTED** [review]
I think this looks good, question on what happens now when a topic is not found.

File: exporter/consumer.go (hunk 1 above)
Comment (kngu9) [reply exporter/consumer.go:189]: Yes, sarama will auto retry until the topic exists and outputs the error.

File: exporter/exporter.go (hunk 2 above)
Comment (kngu9) [reply exporter/exporter.go]: Fixed.

**Review by cmars — COMMENTED** [review]
LGTM with the 100ms exit delay jitter removed; I don't think it's necessary.

**Review by cmars — APPROVED** [review]

**Pull request merged**